packages/data-store/src/typeorm/index.ts

```diff
@@ -0,0 +1,47 @@
+import {DataSource} from 'typeorm';
+import {SDKConfigsEntity} from './entities/sdk-configs.entity';
+import {logger} from '../logger';
+import {DataStoreConfigs} from '../types/types';
+import {NetworkEntity} from './entities/network.entity';
+import {WalletEntity} from './entities/wallet.entity';
+import {DocumentEntity} from './entities/document.entity';
+import {DocumentTypeEntity} from './entities/document-type.entity';
+
+let _dataSource: DataSource;
+
+export async function initializeTypeORM(options: DataStoreConfigs) {
+  let databasePath = 'wallet-data-store.sqlite';
+
+  if (process.env.NODE_ENV === 'test') {
+    databasePath = ':memory:';
+  }
+
+  _dataSource = new DataSource({
+    type: 'sqlite',
+    database: databasePath,
+    entities: [
+      SDKConfigsEntity,
+      NetworkEntity,
+      WalletEntity,
+      DocumentEntity,
+      DocumentTypeEntity,
+    ],
+    // TODO: will remove this once we have all entities in place
+    synchronize: true,
+    dropSchema: options.dropSchema,
+  });
+
+  await _dataSource
```

Comment:
how does closing the datastore get handled? If the app is crashing is there something we can hook into to capture and try to shutdown gracefully?

Reply:
I've added a function to close the connection. Also created a jira to follow up that on the wallet app side. https://dock-team.atlassian.net/browse/DCKA-1625.

The app will need to detect an exist/crash event and call the wallet-sdk function

Reply:
do we need to worry about transaction support so we can do rollbacks or do you think none of our scenarios will need it?